src/main/java/org/broadinstitute/hellbender/engine/filters/ContainsKmerReadFilter.java

```diff
@@ -0,0 +1,38 @@
+package org.broadinstitute.hellbender.engine.filters;
+
+import org.apache.spark.broadcast.Broadcast;
+import org.broadinstitute.hellbender.tools.spark.sv.SVKmerSmall;
+import org.broadinstitute.hellbender.tools.spark.sv.SVKmerizer;
+import org.broadinstitute.hellbender.tools.spark.utils.HopscotchSet;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+import org.junit.Before;
+
+import java.io.Serializable;
+
+/**
+ * Keep reads that contain at least one kmer from a broadcasted HopscotchSet of SVKmerSmall's
+ */
+public class ContainsKmerReadFilter extends ReadFilter implements Serializable {
```

Comment:
Moved class to org.broadinstitute.hellbender.tools.spark.pathseq since this filter is somewhat specialized.